config.toml

```diff
@@ -66,6 +66,7 @@ time_format_blog = "Monday, January 02, 2006"
 description = "Production-Grade Container Orchestration"
 showedit = true
 
+nextMinorVersion = "v1.19"
```

Comment:
Some other ideas (setting/getting state in config.toml):
There is the list of `params.versions`. You could add new fields (next and prev).
Set these values only in the latest version.
Write a shortcode that uses these values from the list.